src/pages/_document.tsx

```diff
@@ -0,0 +1,36 @@
+import Document, { DocumentContext, DocumentInitialProps, Head, Html, Main, NextScript } from 'next/document';
+import React, { ComponentProps } from 'react';
+
+type DocumentProps = ComponentProps<typeof Document>;
+
+const getInitialProps = async (ctx: DocumentContext): Promise<DocumentInitialProps> => {
+  const initialProps = await Document.getInitialProps(ctx);
+  return {
+    ...initialProps,
+  };
+};
+
+const CustomDocument = ({  }: DocumentProps): JSX.Element => {
```

Comment:
`({  }: DocumentProps)`
What does this do? An empty bracket?

Reply:
It means nothing, I'm destructuring the props-object, but not actually picking any of props, since they are not used. The thought process was to keep the typed props in the component if anyone were to ever use any of the props. So that the wouldn't have to look up which props a '_document.tsx' component gets